CommandHandlers.lua

```diff
@@ -0,0 +1,58 @@
+
+
+
+
+
+function HandleLoginCommand(a_Split, a_Player)
+	local PlayerState = GetPlayerState(a_Player)
+	if (PlayerState:IsLoggedIn()) then
+		a_Player:SendMessage("You are already logged in")
+		return true
+	end
+	
+	local Password = a_Split[2]
+	if (not Password) then
+		a_Player:SendMessage("Usage: /login <password>")
+		return true
+	end
+	
+	local res, Err = PlayerState:TryLogin(Password)
+	if (not res) then
+		a_Player:SendMessage(Err)
+		return true
+	end
+	
+	a_Player:SendMessage("You logged in")
+	return true
+end
+
+
+
+
+
+function HandleRegisterCommand(a_Split, a_Player)
+	local PlayerState = GetPlayerState(a_Player)
+	if (PlayerState:IsLoggedIn()) then
+		a_Player:SendMessage("You are already logged in")
+		return true
+	end
+	
+	if ((a_Split[2] == nil) or (a_Split[3] == nil)) then
+		a_Player:SendMessage("Usage: /register <password> <confirmpassword>")
+		return true
+	end
+	
+	local res, Err = PlayerState:TryRegister(a_Split[2], a_Split[3])
```

Comment:
I think the command handler should be responsible for checking that the two passwords match, and the `TryRegister()` function should only receive the single password.
If you ever decide that you don't want double-checking for the password, you can simply remove it in the handler. Or if you want to allow admin to register other people :)